smoke_test/run.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-mpirun -n 1 python -u -O -m mpi4py nozzle.py -i run_params.yaml
+mpirun -n 1 python -u -O -m mpi4py nozzle.py -i run_params.yaml --lazy
```

Comment:
My preference would be to add a lazy-specific smoke test (directory) and run lazy/eager side-by-side in CI.